aten/src/ATen/native/SparseTensorUtils.cpp

```diff
@@ -108,4 +115,15 @@ Tensor coo_to_csr(const int64_t* indices, int64_t dim, int64_t nnz) {
   return csr;
 }
 
+Tensor zeros_with_indices_like(const Tensor& t) {
```

Comment:
Nit: rename `zeros_with_indices_like` to `zeros_like_with_indices` because
- we don't (and likely never will) have `zeros_with_indices`
- and `_with_indices` suffix hints that the function has `zeros_like` behavior with a certain modification.